em++: error: invalid command line setting `-sUSE_ES6_IMPORT_META=0`: Disabling is no longer supported

Hello:

While building h5wasm, I am getting the following error. Can anyone help?

**Error while building:** 
em++: error: invalid command line setting `-sUSE_ES6_IMPORT_META=0`: Disabling is no longer supported

Reference: https://emscripten.org/docs/tools_reference/settings_reference.html
USE_ES6_IMPORT_META: Disabling is no longer supported (Valid values: [1])

**Steps to reproduce:**
1) Clone https://github.com/usnistgov/h5wasm.git
2) make all 

**Emscripten:**
emcc (Emscripten gcc/clang-like replacement + linker emulating GNU ld) 4.0.22 (0f3d2e62bccf8e14497ff19e05a1202c51eb0c65)
clang version 22.0.0git (https:/github.com/llvm/llvm-project c7706d9472fe880ba1d3418919ad4185710c9559)
Target: wasm32-unknown-emscripten
Thread model: posix

**Host OS:**
Linux, ubuntu 24.04

**Build Log:**
[ 25%] Building CXX object CMakeFiles/hdf5_util.dir/src/hdf5_util.cc.o
[ 50%] Building CXX object CMakeFiles/hdf5_util_node.dir/src/hdf5_util.cc.o
[100%] Linking CXX executable /home/taramicro/taramicro/oss-repos/h5wasm/dist/esm/hdf5_util.js
[100%] Linking CXX executable /home/taramicro/taramicro/oss-repos/h5wasm/dist/node/hdf5_util.js
em++: error: invalid command line setting `-sUSE_ES6_IMPORT_META=0`: Disabling is no longer supported
make[3]: *** [CMakeFiles/hdf5_util.dir/build.make:105: /home/taramicro/taramicro/oss-repos/h5wasm/dist/esm/hdf5_util.js] Error 1
make[3]: Leaving directory '/home/taramicro/taramicro/oss-repos/h5wasm/build'
make[2]: *** [CMakeFiles/Makefile2:85: CMakeFiles/hdf5_util.dir/all] Error 2
make[2]: *** Waiting for unfinished jobs....
make[3]: Leaving directory '/home/taramicro/taramicro/oss-repos/h5wasm/build'
[100%] Built target hdf5_util_node
make[2]: Leaving directory '/home/taramicro/taramicro/oss-repos/h5wasm/build'
make[1]: *** [Makefile:91: all] Error 2
make[1]: Leaving directory '/home/taramicro/taramicro/oss-repos/h5wasm/build'
make: *** [Makefile:8: all] Error 2

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -66,7 +66,6 @@ set_target_properties(hdf5_util PROPERTIES
     -s ENVIRONMENT=web,worker \
     -s SINGLE_FILE \
     -s EXPORT_ES6=1 \
-    -s USE_ES6_IMPORT_META=0 \
     -s FORCE_FILESYSTEM=1 \
     -s EXPORTED_RUNTIME_METHODS=\"['ccall', 'cwrap', 'FS', 'AsciiToString', 'UTF8ToString']\" \
     -s EXPORTED_FUNCTIONS=\"${EXPORTED_FUNCTIONS_STRING}\""

